src/app/loadout/loadout-ui/Sockets.tsx

```diff
@@ -25,6 +25,7 @@ interface Props {
   onSocketClick?(
     plugDef: PluggableInventoryItemDefinition,
     /** An allow-list of plug category hashes that can be inserted into this socket */
+    // TODO: why not just pass the socketType hash or socket definition?
```

Comment:
Either type would be fine, I would be inclined to use the socketType hash. The mod picker would then get the SocketType definition from defs and get this list straight from it.

From memory it's done this way as we are looping over sockets and using the whitelist to determine which mods are going in which socket. Since I already had it I passed it along on socket click.